rules/windows/credential_access_iis_apppoolsa_pwd_appcmd.toml

```
@@ -0,0 +1,45 @@
+[metadata]
+creation_date = "2020/08/18"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/18"
+
+[rule]
+author = ["Elastic"]
+description = """
+An attacker with Microsoft IIS webserver access via a webshell or alike can decrypt and dump IIS Appool service account
+password using appcmd command. Verify context of this execution such as parent process, file creation, username to
+validate the legitimacy of this action.
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "lucene"
+license = "Elastic License"
+max_signals = 33
+name = "Microsoft IIS Service Account Password Dumped"
+references = ["https://blog.netspi.com/decrypting-iis-passwords-to-break-out-of-the-dmz-part-1/"]
+risk_score = 73
+rule_id = "0564fb9d-90b9-4234-a411-82a546dc1343"
+severity = "high"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:process and event.type:(start or process_started) and
+  (process.name:appcmd.exe  or process.pe.original_file_name:appcmd.exe) and
+  process.args:(/[lL][iI][sS][tT]/ and //[tT][eE][xX][tT]\:[pP][aA][sS][sS][wW][oO][rR][dD]/)
```

```suggestion
event.category:process AND event.type:(start OR process_started) AND
  (process.name:appcmd.exe  OR process.pe.original_file_name:appcmd.exe) AND
  process.args:(/[lL][iI][sS][tT]/ AND //[tT][eE][xX][tT]\:[pP][aA][sS][sS][wW][oO][rR][dD]/)
```

lucene requires uppercase